src/post/post.entity.ts

```diff
@@ -106,9 +107,29 @@ export class Post extends BaseEntity {
         if (!account.id) {
             throw new Error('Cannot add like for account with no id');
         }
+        this.likesToRemove.delete(account.id);
         this.potentiallyNewLikes.add(account.id);
     }
 
+    removeLike(account: Account) {
+        if (!account.id) {
+            throw new Error('Cannot add like for account with no id');
+        }
+        this.potentiallyNewLikes.delete(account.id);
+        this.likesToRemove.add(account.id);
+    }
+
+    getChangedLikes() {
+        const likesToRemove = [...this.likesToRemove.values()];
+        this.likesToRemove.clear();
+        const likesToAdd = [...this.potentiallyNewLikes.values()];
+        this.potentiallyNewLikes.clear();
```

Comment:
We can rename `potentiallyNewLikes` to `likesToAdd` as we did in [post.repository.knex.ts](https://github.com/TryGhost/ActivityPub/pull/336/files?diff=split&w=1#diff-80d34d2729ea70828740b1b3a8b31097755c1c8c0d4443102b68ce637814ec6bR103), to maintain consistent naming.

Reply:
Yeah I agree, I'll do this in a cleanup commit